src/main/java/seedu/address/model/lesson/Date.java

```diff
@@ -33,6 +34,7 @@
 
     public final String value;
 
+    private final LocalDate localDate;
```

Comment:
`localDate` is not very clear what this variable represents eh, if I am not wrong I think this is the recurring lesson date which we use to update? If that's the case should we have a more descriptive name like `nextLessonDate` or something to that extent?

Reply:
oh icic okok